.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      
+    - name: Install dependencies
+      run: make
+      
```

Comment:
Since `make test` installs dependencies, no need to have these lines. Only checkout & run tests steps are enough.

Reply:
done